apps/user-office-frontend/src/components/user/UpdateUserInformation.tsx

```diff
@@ -164,40 +159,6 @@ export default function UpdateUserInformation(
     );
   };
 
-  const isUserOfficer = currentRole === UserRole.USER_OFFICER;

```

Comment:
So all this thing with the placeholder and email verification is not needed anymore. What about the invitation codes? Aren't those inviting user as a placeholder or it has another way of handling?